src/migrateContentServer.test.ts

```diff
@@ -336,4 +395,87 @@ describe("migrateContentServer", () => {
     // The filters in 5.0 are represented as strings.
     expect(typeof migratedWidgetFilter.mdx).toBe("string");
   });
+
+  it("migrates the KPI custom titles while dropping custom titles that could not be successfully migrated", async () => {
+    const legacyUIFolderWithInvalidKpiTitle = addLegacyBookmarkToUIFolder(
```

Comment:
Here too, it would be nice to have dedicated test resources for:
- `legacyKPIWithInvalidTitle`
- `legacyUIFolderWithKpiWithInvalidTitle`

Reply:
Done, these got split out into `__test_resources__/legacyKpiWithInvalidTitle.ts` and `__test_resources__/legacyUIFolderWithKpiWithInvalidTitle.ts`